lib/closex/mock_client/mock_client.ex

```diff
@@ -30,6 +30,9 @@ defmodule Closex.MockClient do
   @not_found_id "not_found"
   def not_found_id, do: @not_found_id
 
+  @not_found_query "Jasdeep singh ptjasdeepsingh@gmail.com"
```

Comment:
is this a real email address?

Reply:
I would maybe vote we dont use `.gmail` in case it is an actual email address, is that possible?

Reply:
It's a valid one but may also be a real one - hence what I was asking in my PR description about anonymisation. Could be a real guy, but if it isn't I'd rather not change it because it's all over our elixir projects at the moment.

Reply:
Okay yea thats fair man no wrries